File: chatwoot/python/src/GetCsatSurveyPageExample.py
import json
from datetime import date, datetime
from pprint import pprint

from chatwoot_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
)

with ApiClient(configuration) as api_client:
    try:
        api.CSATSurveyPageApi(api_client).get_csat_survey_page(
            conversation_uuid=0,
        )
    except ApiException as e:
        print("Exception when calling CSATSurveyPageApi#get_csat_survey_page: %s\n" % e)

File: chatwoot/ruby/src/GetCsatSurveyPageExample.rb
require "json"
require "chatwoot_client"

ChatwootClient.configure do |config|
end

begin
    ChatwootClient::CSATSurveyPageApi.new.get_csat_survey_page(
        0, # conversation_uuid
    )
rescue ChatwootClient::ApiError => e
    puts "Exception when calling CSATSurveyPageApi#get_csat_survey_page: #{e}"
end
